README.rst

```diff
@@ -62,10 +61,6 @@ project so conditional dependencies that require a specific Python version,
 or other environment markers, resolve relative to your project's
 environment.
 
-**Note**: ensure you don't have ``requirements.txt`` if you compile
-``setup.py`` or ``requirements.in`` from scratch, otherwise, it might
-interfere.
-
```

Comment:
Could you elaborate on these deletions above? There is a reason for this note, see https://github.com/jazzband/pip-tools/issues/744.

Reply:
I found "from scratch" and "might interfere" very vague ("from scratch" in what sense? interfere how? why only "might"?).

I think the new text covers how an existing requirements.txt affects pip-compile better in the sense that explains what will happen more precisely and completely. But I'm happy to restore it if you prefer?

Reply:
> I found "from scratch" and "might interfere" very vague ("from scratch" in what sense? interfere how? why only "might"?).

I'd prefer to keep the note. I like the explanation in the "Updating requirements" section.  Would you like to improve the note somehow?

Reply:
How about "Note: To compile setup.py or requirements.in from scratch, you must first delete any existing requirements.txt file"?

Reply:
That's a good direction. Perhaps we should clarify shortly why `requirements.txt` must be deleted.

Reply:
Something like this (basically copy/pasted from the new text below)?

"Note: If ``pip-compile`` finds an existing ``requirements.txt`` file, and that file fulfils the dependencies, then no changes will be made, even if updates are available. To compile setup.py or requirements.in from scratch, first delete any existing requirements.txt file, or see "Updating requirements" for alternative approaches."

Reply:
Pushed another revision. Please let me know if this is more along the right lines.

Reply:
Thanks! Looks good 👍🏻